corehq/apps/sms/change_publishers.py

```diff
@@ -1,10 +1,20 @@
 from corehq.apps.change_feed import topics
```

Comment:
do you mean making the publish an async task?  my thoughts around that were just to not make the sms processing have to wait for that, and also to let it retry over a longer period of time in case kafka goes down.

i also happened to notice: https://github.com/dimagi/commcare-hq/blob/e086a5eae0a68826c9e73ca0675af02124a086db/corehq/apps/change_feed/producer.py#L34 which i didn't want to cause a slowdown in case something with kafka goes wrong